### ISSUE TITLE
Color contrast of keyboard focus on "Go to bikesharingweb" with respect to background is 1.1 which is less than 3.1.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode#128047]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_VSCode;#A11ySev2;#A11yTCS;#A11yWCAG2.1;#DesktopApp;#Visual Studio Code Client;#WCAG1.4.11;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#A11yMAS;#FTP;#ColorContrastAnalyzer:#ColorContrast;

### Environment Details:
Application Name: Bridge to Kubernetes (Mindaro)
Windows Version: Windows10

### Repro Steps:

1. Open visual studio code and load "Bikes" project.
2. Apply Light+ (default light) color theme
3. Tab to the terminal and type command "npm i",
4. Tab to  extensions panel and install "Local Process with Kubernetes" extension.
5. Tab till Kubernetes menu from left navigation panel and press enter.
6. Tab to Set Kubeconfig option from more menu item and press enter to select it
7. Tab to “+ Add new kubeconfig” then tab to “AccessibilityTesting-Kubeconfig” and select it.
8.  Tab till bottom of the VS Code window adn press enter on this “Kubernetes: bikeapp (Accessibility Testing-Cluster)” item
9.  A Search panel should open.
10. Tab to any listitem
11. Open color contrast analyzer and check the colour contrast of keyboard focus with respect to grey background

### Actual:
Color contrast of keyboard focus on "Go to bikesharingweb" with respect to background is 1.1 which is less than 3.1.

**Similar issue is observed throughout the application where dropdown is there**

### Expected:
Color contrast of keyboard focus on "Go to bikesharingweb" with respect to background should be more than 3.1

### User Impact:
Low vision users will face problem in navigating if keyboard focus will not be visible

**Comment by misolori** [comment]
@Yash14j can you please share a screenshot?

**Comment by fsteffi** [comment]
Bridge to Kubernetes is the extension  @misolori
